README.md

```diff
@@ -34,6 +33,15 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Options for js/css tasks
+
+`--minify` | Render additional minified files, suffixed with `.min`. Will be `true` in production builds.
```

Comment:
I was [looking at the readme](https://github.com/stefanpl/gulp-task-collection/tree/fix/working-production-build#options-for-jscss-tasks) to read through the options and it looks like the formatting could do with being changed here. 

Maybe this can be a table and also include columns for values and default values

Reply:
I think we can optimize the readme in a separate PR. The proposed changes here are improving the docs a lot already.

Reply:
- Concat is now `false` by default
- Improved the readme to include an actual table